tests/unit/intake_sqlite_test.py

```diff
@@ -0,0 +1,17 @@
+"""A dummy unit test so pytest has something to do."""
+import logging
+from pathlib import Path
+
+from intake_sqlite import urlpath_to_sqliteurl
+
+logger = logging.getLogger(__name__)
+
+TEST_DIR = Path(__file__).parent.parent.resolve()
+
+
+def test_urlpath_to_sqliteurl() -> None:
+    """Test our transformation of paths/URLs into SQL Alchemy URLs."""
+    expected_local_url = "sqlite:///" + str(TEST_DIR / "data/test.db")
+    test_db_path = TEST_DIR / "data/test.db"
```

Comment:
Should there be another test for remote URLs?

Reply:
Definitely. I just wanted some nominal tests. I have a checklist of additional tests to add in the PR description.